android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerSimpleCache.kt

```diff
@@ -0,0 +1,31 @@
+package com.brentvatne.exoplayer
+
+import android.content.Context
+import androidx.media3.database.StandaloneDatabaseProvider
+import androidx.media3.datasource.DataSource
+import androidx.media3.datasource.HttpDataSource
+import androidx.media3.datasource.cache.CacheDataSource
+import androidx.media3.datasource.cache.LeastRecentlyUsedCacheEvictor
+import androidx.media3.datasource.cache.SimpleCache
+import java.io.File
+
+object RNVSimpleCache {
```

Comment:
The cache instance could also be a property of ReactExoplayerViewManager as there is only 1 viewmanager per module. And passed to ReactExoplayerView in createViewInstance.

Just my 2 cents. Can't wait for the merge to be accepted.

Reply:
thats fair; i have no idea the viewManager is only initialized once, if u'd like to make a commit please go ahead